salt/cloud/clouds/hetzner.py

```diff
@@ -83,13 +83,19 @@ def _datacenter_to_dict(datacenter):
     }
 
 
-def _network_to_dict(net):
+def _public_network_to_dict(net):
     return {
         "ipv4": getattr(net.ipv4, "ip", None),
         "ipv6": getattr(net.ipv6, "ip", None),
     }
 
 
+def _private_network_to_dict(net):
+    return {
+        "ipv4": getattr(net.ip, "ip", None),
```

Comment:
Is there not an instance where private will have an ipv6 address?

Reply:
Hi! No. Private nets are always represented as IPv4 in the Hetzner Cloud API. 

https://docs.hetzner.cloud#servers-get-a-server